jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListSavedFiltersPresenter.java

```diff
@@ -0,0 +1,158 @@
+import org.jbpm.workbench.df.client.filter.FilterSettings;
+import org.jbpm.workbench.df.client.filter.FilterSettingsBuilderHelper;
+import org.jbpm.workbench.es.client.i18n.Constants;
+import org.uberfire.client.annotations.WorkbenchMenu;
+import org.uberfire.client.annotations.WorkbenchScreen;
+import org.uberfire.workbench.model.menu.Menus;
+
+import static org.dashbuilder.dataset.filter.FilterFactory.equalsTo;
+import static org.dashbuilder.dataset.sort.SortOrder.DESCENDING;
+import static org.jbpm.workbench.common.client.PerspectiveIds.EXECUTION_ERROR_LIST_SAVED_FILTERS_SCREEN;
+import static org.jbpm.workbench.es.model.ExecutionErrorDataSetConstants.*;
+
+@Dependent
+@WorkbenchScreen(identifier = EXECUTION_ERROR_LIST_SAVED_FILTERS_SCREEN)
+public class ExecutionErrorListSavedFiltersPresenter extends SavedFiltersPresenter {
+
+    private static final String EXECUTION_ERROR_LIST_PREFIX = "DS_ExecutionErrorListGrid";
+    protected static final String TAB_ALL = EXECUTION_ERROR_LIST_PREFIX + "_0";
+    protected static final String TAB_ACK = EXECUTION_ERROR_LIST_PREFIX + "_1";
+    protected static final String TAB_NEW = EXECUTION_ERROR_LIST_PREFIX + "_2";
+
+    private Constants constants = Constants.INSTANCE;
+
+    @WorkbenchMenu
+    public Menus getMenus() { //It's necessary to annotate with @WorkbenchMenu in subclass
+        return super.getMenus();
+    }
+
+    @Override
+    public String getGridGlobalPreferencesKey() {
+        return EXECUTION_ERROR_LIST_PREFIX;
+    }
+
+    /*-------------------------------------------------*/
+    /*---              DashBuilder                   --*/
+    /*-------------------------------------------------*/
+    @Override
+    public FilterSettings createTableSettingsPrototype() {
+        FilterSettingsBuilderHelper builder = FilterSettingsBuilderHelper.init();
+        builder.initBuilder();
+        builder.dataset(EXECUTION_ERROR_LIST_DATASET);
+
+        builder.filterOn(true,
+                         true,
+                         true);
+        builder.tableOrderEnabled(true);
+        builder.tableOrderDefault(COLUMN_ERROR_DATE,
+                                  DESCENDING);
+        builder.tableWidth(1000);
+
+        return builder.buildSettings();
+    }
+
+    private FilterSettings createFilterTabSettings(final Boolean acknowledged) {
+        FilterSettingsBuilderHelper builder = FilterSettingsBuilderHelper.init();
+        builder.initBuilder();
+
+        builder.dataset(EXECUTION_ERROR_LIST_DATASET);
+
+        if (acknowledged != null) {
+            builder.filter(equalsTo(COLUMN_ERROR_ACK,
+                                    acknowledged ? "1" : "0"));
+        }
+
+        builder.filterOn(true,
+                         true,
+                         true);
+        builder.tableOrderEnabled(true);
+        builder.tableOrderDefault(COLUMN_ERROR_DATE,
+                                  DESCENDING);
+        addCommonColumnSettings(builder);
```

Comment:
All other presenters except this one lack a separate method for setting columns only. Maybe we should make it a common thing?